server/map_rpc_server.go

```diff
@@ -148,10 +148,7 @@ func (t *TrillianMapServer) getLeavesByRevision(ctx context.Context, mapID int64
 			found++
 		} else {
 			// Empty leaf for proof of non-existence.
-			leafHash, err := hasher.HashLeaf(mapID, index, nil)
-			if err != nil {
-				return nil, fmt.Errorf("HashLeaf(nil): %v", err)
-			}
+			leafHash := hasher.HashEmpty(mapID, index, 0)
```

Comment:
I think we decided to return "" for this case, in which we can just drop this line.

We should probably leave a comment to that effect though

Reply:
Yeah, it's gone with the update to plan A revision 2.